Tests/Common/Orders/Fees/BybitFuturesFeeModelTests.cs

```diff
@@ -0,0 +1,221 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using NUnit.Framework;
+
+using QuantConnect.Brokerages;
+using QuantConnect.Data.Market;
+using QuantConnect.Orders;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Securities;
+using QuantConnect.Tests.Brokerages;
+using QuantConnect.Securities.CryptoFuture;
+using QuantConnect.Util;
+
+namespace QuantConnect.Tests.Common.Orders.Fees
+{
+    [TestFixture]
+    public class BybitFuturesFeeModelTests
```

Comment:
Believe some of these unit tests are failing in the CI

Reply:
I was checking the log but couldn't find the exact place, are there any logs available? Seems like this is a case of "works on my machine" 🙈 
![image](https://github.com/QuantConnect/Lean/assets/18667020/c8a8b0ee-c733-4881-81c3-ec93ddb6136e)

Reply:
> Seems like this is a case of "works on my machine" 🙈

Classic 😄. Yeah, can see the logs here https://github.com/QuantConnect/Lean/actions/runs/6162742989/job/16734280623?pr=7442 (or raw https://pipelinesghubeus22.actions.githubusercontent.com/AkHYsUuSQonYCaj3O9X6HfU0sjL2p7Bziw0TViY1DkKAki8r1B/_apis/pipelines/1/runs/27173/signedlogcontent/2?urlExpires=2023-09-15T17%3A41%3A02.7737175Z&urlSigningMethod=HMACV1&urlSignature=iNFDeEvostyXZrteiPrYWQLpEcFzmYq0tOlvubJ2lRQ%3D) 
<img width="877" alt="image" src="https://github.com/QuantConnect/Lean/assets/18473240/af1af642-8ab4-4b34-9345-219e695515ec">

Reply:
Thank you! And I was wrong.. was more like a bug in my IDE just showing them green.